src/components/TodoContainer.jsx

```diff
@@ -54,15 +52,23 @@ export default function TodoContainer() {
     };
 
     try {
-      const res = await fetch(url, options);
+      const res = await fetch(
+        `${url}?view=Grid%20view&sort[0][field]=title&sort[0][direction]=asc`,
+        options
+      );
 
       if (!res.ok) {
         const message = `Error: ${res.status}`;
         throw new Error(message);
       }
 
       const data = await res.json();
-      const todos = data.records.map((todoObject) => {
+
+      const sortedData = data.records.sort((a, b) =>
+        a.fields.title.toLowerCase() > b.fields.title.toLowerCase() ? 1 : -1
```

Comment:
This would also work.
``a.fields.title.toLowerCase().localeCompare(b.fields.title.toLowerCase())``